docs/templating/twig_function_reference/site_context_twig_functions.md

```diff
@@ -0,0 +1,21 @@
+---
+description: Site context Twig function determines if given location is site context-aware.
+edition: experience
+page_type: reference
+---
+
+# Site context Twig functions
+
+You can use site context Twig functions to determine if given location is site context-aware.
```

Comment:
It's technically a [Twig test](https://twig.symfony.com/doc/3.x/tests/index.html), so maybe:
```suggestion
You can use site context Twig test to determine if given location is site context-aware.
```
Not sure if it makes sense, but we can link to the Twig documentation as above.